ent/schema/price.go

```diff
@@ -46,6 +46,28 @@ func (Price) Fields() []ent.Field {
 				"postgres": "varchar(255)",
 			}).
 			NotEmpty(),
+		field.String("custom_pricing_unit_id").
+			SchemaType(map[string]string{
+				"postgres": "varchar(50)",
+			}).
+			Optional(),
+		field.Float("price_unit_amount").
+			SchemaType(map[string]string{
+				"postgres": "numeric(25,15)",
+			}).
+			Optional(),
+		field.String("display_price_unit_amount").
+			SchemaType(map[string]string{
+				"postgres": "varchar(255)",
+			}).
+			Optional(),
```

Comment:
Let's remove this

Reply:
all customer_pricing_unit can have same prefix else it is confusing
weather the conversion rate is for custom pricing unit or something

can try:
price_unit_*

price_unit_id
price_unit_code
price_unit_amount
price_unit_conversion_factor

can just keep these fields maybe and remove others and pull them from pricing_unit table and enrich in the DTO if required. Something like:
```json
.....existing fields
"custom_pricing_unit": {
entire custom pricing response DTO
}
.....existing fields
```


can follow this exact pattern for subscription line items and invoice line items as well
in the LIST APIs of these entities can add a expand param `pricing_unit` which, when passed will expand the params for all the results.